app/internal/emotion.py

```diff
@@ -0,0 +1,92 @@
+from collections import namedtuple
+
+import text2emotion as te
+from typing import Dict, Tuple, Union
+
+from app.config import (
+    CONTENT_WEIGHTS,
+    LEVEL_OF_SIGNIFICANCE,
+    TITLE_WEIGHTS)
+
+
+EMOTIONS = {"Happy": "&#128515",
+            "Sad": "&#128577",
+            "Angry": "&#128544",
+            "Fear": "&#128561",
+            "Surprise": "&#128558"}
+
+Emoticon = namedtuple("Emoticon",
+                      ["dominant_emotion", "emotion_score", "emoticon_code"])
+
+
+def get_weights(emotion: str, title_emotion: Dict[str, float],
+                content_emotion: Dict[str, float] = None) -> float:
+    if not content_emotion:
+        return title_emotion[emotion]
+    return (title_emotion[emotion] * TITLE_WEIGHTS +
+            content_emotion[emotion] * CONTENT_WEIGHTS)
+
+
+def score_comp(emotion_score: float, DominantEmotion: Emoticon, emotion: str,
+               code: str, flag: int) -> Tuple[Emoticon, int]:
+    """
+    score comparison between emotions.
+    returns the dominant and if equals we flag it
+    """
+    if emotion_score > DominantEmotion.emotion_score:
+        flag = 0
+        DominantEmotion = Emoticon(emotion, emotion_score, code)
+    elif emotion_score == DominantEmotion.emotion_score:
+        flag = 1
+    return (DominantEmotion, flag)
```

Comment:
What if emotion_score < DominantEmotion.emotion_score?

Reply:
the DominantEmotion will not change